powerauth-test-server/DockerfileExtended

```diff
@@ -0,0 +1,54 @@
+FROM tomcat:jdk11-adoptopenjdk-openj9
```

Comment:
I am not sure we should use `powerauth-test-server/DockerfileExtended`. Dockerfiles are always called `Dockerfile`.

Reply:
Finally, this file will be deleted.